mitodl/ocw-to-hugo
# add thumbnail image

#### Pre-Flight checklist


#### What are the relevant tickets?
https://trello.com/c/qiCvN0U8/90-add-thumbnail-to-mobile-course

#### What's this PR do?
1) Adds `course_thumbnail_image_url ` to the hugo markdown

#### How should this be manually tested?
Download dropbox course link with parsed ocw data which includes  `thumbnail_image_src`  in the master json.

Run the ocw to hugo scrip with the new course data. Verify that the script runs correctly and the hugo markdown includes `course_thumbnail_image_url`
